### PR TITLE
Changes to support configurable healthport in node

### PR DESCRIPTION
**Pull request by jadech32**
## Description

- Added environment variable `FELIX-HEALTHPORT`
- Changed liveness check for felix from http request to command. 

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit ccf0c2b5d0644f009a95a769f1bd452e48a15a15

https://deploy-preview-2634--calico.netlify.com

File: _includes/master/charts/calico/templates/calico-node.yaml [hunk 1]
```diff
@@ -271,6 +271,9 @@ spec:
               value: "info"
             - name: FELIX_HEALTHENABLED
               value: "true"
+            # Set default felix liveness and readiness health-check port
```
Comment (caseydavenport) [inline _includes/master/charts/calico/templates/calico-node.yaml]: This is the default value, right? If so, we probably shouldn't need to specify it here.

**Comment by caseydavenport** [comment]
@jadech32 if you could remove the FELIX_HEALTHPORT env var and rebase I'd like to merge this PR :)

**Comment by jadech32** [comment]
@caseydavenport Done!

**Pull request merged**